TESTS/integration/README.md

```diff
@@ -0,0 +1,96 @@
+
+| **Test suite** | **Description** |
+| ------------- | ------------- |
+| `fs-single` | File system single-threaded tests with write buffer sizes - 1 byte, 4b, 16b, 64b, 256b, 1kb, 4kb, 16kb, 64kb. |
+| `fs-multi` | File system multithreaded test with write buffer sizes - 1 byte, 4b, 16b, 64b, 256b, 1kb, 4kb, 16kb, 64kb. |
+| `net-single` | Network single-threaded test with receive buffer sizes - 128 bytes, 256b, 1kb, 2kb, 4kb. |
+| `net-multi` | Network multithreaded test for 1, 2 and 3 download threads with 1kb receive buffer size. |
+| `stress-net-fs` | Network and file system single and multithreaded tests:<ul><li>memory allocation - 10k, 20k, 40k, 60k</li><li>1 thread (sequential) - 1 download (1kb buffer), 1 file thread (1kb buffer)</li><li>2 parallel threads - 1 download, 1 file thread (1kb buffer)</li><li>3 parallel threads - 1 download, 2 file (256 bytes, 1 kb buffer)</li><li>4 parallel threads - 1 download, 3 file (1 byte, 256 bytes, 1kb buffer)</li></ul> |
+
+
+
+## Execute tests
+
+The integration test requires both the connectivity and and storage device to to configured. Currently we have a pre-configuration file defined in the COMMON folder, and that file covers some of the platform connectivity/storage combinations.  If you have your own platform or you want to define your own configuration, please follow the [configuration guide](#Tests-configuration).
+
+To compile and run the tests inside mbed-os directory, the configuration file need to be passed in the command line: 
+
+    ```bash
+    $ mbed test -t <toolchain> -m <platform> -n *integraion-* --app-config TESTS\integration\COMMON\mbed_app.json
+    ```
+
+### table of platforms in the configuration file
+
+
+Platform                        |  Connectivity      | Storage 
+--------------------------------| -------------------| --------- 
+Nuvoton NUMAKER-IOT-M487            | Wi-Fi              | SD Card 
+Nuvoton NUMAKER-PFM-M487            | Ethernet           | SD Card 
+Nuvoton NUMAKER-PFM-NUC472          | Ethernet           | SD Card 
+NXP FRDM-K64F                       | Ethernet           | SD Card 
+NXP FRDM-K66F                       | Ethernet           | SD Card 
+Renesas GR-PEACH                    | Ethernet           | SD Card 
+Renesas GR-LYCHEE                   | Wi-Fi              | SD Card
+Sigma Delta Technologies SDT64B     | Ethernet           | SD Card 
+Silicon Labs Giant Gecko 11         | Ethernet           | QSPI 
+Silicon Labs Thunderboard Sense 2   | 15.4 Thread        | SPI Flash
+ST DISCO_L475E_IOT01A               | Wi-Fi              | QSPI 
+ST DISCO_F413H                      | Wi-Fi              | QSPI 
+ST DISCO_F469NI                     | Wi-Fi Shield       | QSPI
+ST DISCO_F746NG                     | Ethernet           | QSPI
+ST DISCO_F769NI                     | Ethernet           | QSPI
+ST DISCO_L496AG                     | Wi-Fi Shield       | QSPI
+ST NUCLEO_F207ZG                    | Ethernet           | SD Card
```

Comment:
Let's remove configurations that require a shield for now ;-)

Reply:
listing platforms in a readme can get stale quickly.  Its another thing to keep updated.  is it necessary?

Reply:
Agree with Mac, unless readme can provide a link to a markdown target doc that is autogenerated or some other solution using a single source of Truth

Reply:
I think the table in the README is just mean to give users some hint for what we tested, and know for working. But yes it is tricky to keep the table updated